src/main/java/mil/dds/anet/search/mssql/MssqlAuthorizationGroupSearcher.java

```diff
@@ -1,97 +1,39 @@
-		if (doFullTextSearch) {
-			// If we're doing a full-text search, add a pseudo-rank (the sum of all search ranks)
-			// so we can sort on it (show the most relevant hits at the top).
-			// Note that summing up independent ranks is not ideal, but it's the best we can do now.
-			// See https://docs.microsoft.com/en-us/sql/relational-databases/search/limit-search-results-with-rank
-			sql.append(", ISNULL(c_authorizationGroups.rank, 0) + ISNULL(f_authorizationGroups.rank, 0)");
-			sql.append(" AS search_rank");
-		}
-		sql.append(", count(*) over() as totalCount FROM authorizationGroups");
-
-		if (doFullTextSearch) {
-			sql.append(" LEFT JOIN CONTAINSTABLE (authorizationGroups, (name, description), :containsQuery) c_authorizationGroups"
-					+ " ON authorizationGroups.id = c_authorizationGroups.[Key]"
-					+ " LEFT JOIN FREETEXTTABLE(authorizationGroups, (name, description), :freetextQuery) f_authorizationGroups"
-					+ " ON authorizationGroups.id = f_authorizationGroups.[Key]");
-			whereClauses.add("c_authorizationGroups.rank IS NOT NULL");
-			sqlArgs.put("containsQuery", Utils.getSqlServerFullTextQuery(text));
-			sqlArgs.put("freetextQuery", text);
-		}
-
-		if (query.getStatus() != null) {
-			whereClauses.add("authorizationGroups.status = :status");
-			sqlArgs.put("status", DaoUtils.getEnumId(query.getStatus()));
-		}
-
-		final AuthorizationGroupList result = new AuthorizationGroupList();
+		AuthorizationGroupList result = new AuthorizationGroupList();
 		result.setPageNum(query.getPageNum());
 		result.setPageSize(query.getPageSize());
-
-		if (whereClauses.isEmpty()) {
+		if (query.getText() == null || query.getText().trim().length() == 0) {
 			return result;
 		}
 
-		sql.append(" WHERE ");
-		sql.append(Joiner.on(" AND ").join(whereClauses));
-
-		//Sort Ordering
-		final List<String> orderByClauses = new LinkedList<>();
-		if (doFullTextSearch && query.getSortBy() == null) {
-			// We're doing a full-text search without an explicit sort order,
-			// so sort first on the search pseudo-rank.
-			orderByClauses.addAll(Utils.addOrderBy(SortOrder.DESC, null, "search_rank"));
-		}
-
-		if (query.getSortBy() == null) { query.setSortBy(AuthorizationGroupSearchSortBy.NAME); }
-		if (query.getSortOrder() == null) { query.setSortOrder(SortOrder.ASC); }
-		switch (query.getSortBy()) {
-			case CREATED_AT:
-				orderByClauses.addAll(Utils.addOrderBy(query.getSortOrder(), "authorizationGroups", "createdAt"));
-				break;
-			case NAME:
-			default:
-				orderByClauses.addAll(Utils.addOrderBy(query.getSortOrder(), "authorizationGroups", "name"));
-				break;
-		}
-		orderByClauses.addAll(Utils.addOrderBy(SortOrder.ASC, "authorizationGroups", "id"));
-		sql.append(" ORDER BY ");
-		sql.append(Joiner.on(", ").join(orderByClauses));
+		final Map<String,Object> args = new HashMap<String,Object>();
+		final StringBuilder sql = new StringBuilder(
+				"/* MssqlLocationSearch */ SELECT *, count(*) over() as totalCount "
```

Comment:
Comment should not be `MssqlLocationSearch` but `MssqlAuthorizationGroupSearch`.

Reply:
fixing that and also making the status to work.